integration/simulation/params/wallet_utils.go

```diff
@@ -0,0 +1,66 @@
+package params
+
+import (
+	"math/big"
+
+	"github.com/obscuronet/obscuro-playground/go/obscuronode/enclave/evm"
+	"github.com/obscuronet/obscuro-playground/go/obscuronode/wallet"
+	"github.com/obscuronet/obscuro-playground/integration"
+	"github.com/obscuronet/obscuro-playground/integration/datagenerator"
+)
+
+type SimWallets struct {
+	MCOwnerWallet        wallet.Wallet
```

Comment:
@tudor-malene Can this have a better name or a comment? Making me think of a big mac.